library/src/interactor_impl.cxx

```diff
@@ -193,7 +198,16 @@ class interactor_impl::internals
         self->Window.PrintSceneDescription(log::VerboseLevel::INFO);
         break;
       default:
-        if (keySym == "Left")
+
+        if (keyCode == '2' || keySym == "KP_Down")
+        {
+          self->VTKInteractor->SetInteractorStyle(self->Style2D);
```

Comment:
does that do anything with trackball on/off ?

Reply:
Not sure what you mean. When enabling the 2d style, the trackball (ie Style3D) is disabled.

Reply:
ok, just checking.